catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/ModalRegistry.view.js

```diff
@@ -45,8 +45,9 @@ function (ich,Marionette,Backbone,ConfigurationEdit,wreqr,_,$,Utils,Service,moda
         className: 'modal',
         events: {
             "click .submit-button": "submitData",
-            "click .cancel-button": "closeAndUnbind",
-            "change .registryName": "registryNameChanged"
+            "click .cancel-button": "cancel",
+            "change .registryName": "registryNameChanged",
+            "change #registryUrl" : "urlChanged"
```

Comment:
The registryUrl element is generated by a handlebars template on the registry modal. If you are running ddf and open the registryModal, you will find the registryUrl element.